template/private-npm-package/skuba.template.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Run `skuba configure` to finish templating and remove this file.
+ */
+
+module.exports = {
+  entryPoint: 'src/index.ts',
+  fields: [
+    {
+      name: 'moduleName',
+      message: 'Module name',
+      initial: 'module',
```

Comment:
Given this is the _private_ template, I wonder if this should be:

```suggestion
      initial: '@seek/my-first-module',
```

And we could even add a `validate` function here:

```typescript
(value) => /^@seek\/.+$/.test(value) || `module must start with an ${chalk.bold('@seek/')} scope`
```